ajax/ticketsatisfaction.php

```diff
@@ -98,6 +103,46 @@
         );
         echo "</td></tr>";
 
+        echo "<tr class='tab_bg_1'><td width='50%'>" . __('Max rate') . "</td>";
+        echo "<td>";
+        Dropdown::showNumber(
+            'inquest_max_rate',
+            ['value' => $inquest_max_rate,
+                'min'   => 1,
+                'max'   => 5,
+                'step'  => 1,
+                'unit'  => ''
+            ]
+        );
+        echo "</td></tr>";
+
+        echo "<tr class='tab_bg_1'><td width='50%'>" . __('Default rate') . "</td>";
+        echo "<td>";
+        Dropdown::showNumber(
+            'inquest_default_rate',
+            ['value' => $inquest_default_rate,
+                'min'   => 1,
+                'max'   => 5,
+                'step'  => 1,
+                'unit'  => ''
+            ]
+        );
+        echo "</td></tr>";
+
+        echo "<tr class='tab_bg_1'><td width='50%'>" . __('Comment required if score is less than or equal to') . "</td>";
```

Comment:
```suggestion
        echo "<tr class='tab_bg_1'><td width='50%'>" . __('Comment required if score is <= to') . "</td>";
```

maybe we can use a symbol instead a full wording here ?
your opinion ?

Reply:
I think most people will understand that